README.md

```diff
@@ -268,6 +305,43 @@ and replace it with '/public/'.
 
 _Default_: `''`
 
+#### runtimeCaching [Array&#x27e8;Object&#x27e9;]
+Used to configure runtime caching for dynamic content. If you use this option, the `sw-toolbox`
+library configured with the caching strategies you specify will automatically be included in
+your generated service worker file.
+
+Each `Object` in the `Array` needs a `urlPattern`, which is either a
+[`RegExp`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp)
+or a string, following the conventions of the `sw-toolbox` library's
+[routing configuration](https://github.com/GoogleChrome/sw-toolbox#basic-routes). Also required is
+a `handler`, which should be either a string corresponding to one of the
+[built-in handlers](https://github.com/GoogleChrome/sw-toolbox#built-in-handlers) under the `toolbox.` namespace, or a function corresponding to your custom
+[request handler](https://github.com/GoogleChrome/sw-toolbox#defining-request-handlers). There is also
+support for `options`, which corresponds to the same options supported by a
+[`sw-toolbox` handler](https://github.com/GoogleChrome/sw-toolbox#options).
+
+For example, the following defines runtime caching behavior for two different URL patterns, using a
+different handler for each, and specifying a dedicated cache with maximum size for requests
+matching `/articles/`:
+
```

Comment:
That's a little much for one sentence. Try this:

"For example, the following defines runtime caching behavior for two different URL patterns. It uses a
 +different handler for each, and specifies a dedicated cache with maximum size for requests
 +that match `/articles/`:"